Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
```

Comment:
You're missing an entire translator info section. Please use [Scaffold](https://www.zotero.org/support/dev/translators/scaffold) to create translators, then commit the complete file that Scaffold generates. See the top of this translator for what I'm referring to https://github.com/zotero/translators/blob/master/3news.co.nz.js